### DISCUSSION TITLE
[FEATURE] Image Processing Component

### DISCUSSION THREAD

**Discussion by SurajRaika** [writer/writer-framework#128] [Ideas]
### Description
Develop an image processing component within the "streamsync" project that enables users to select an image. The selected image will be transmitted to the backend for applying computer vision filters. Once processed, the modified image will be returned to the frontend for display.

### Reference Image

![Reference Image](https://github.com/streamsync-cloud/streamsync/assets/103476230/4faec3a5-8729-4979-b4c5-0c999a4268a6)

**Comment by ramedina86** [comment]
Hi, what would be the advantage of this over a standard file upload and an _Image_ component? I agree that the file upload component does need to be improved though, and perhaps show thumbnails as you upload. But I'm not convinced we need a dedicated Image Processing component. Interested in hearing your thoughts though.

**Comment by ramedina86** [comment]
Also, there are some pressing issues (e.g. bugs) on the Issues section so I've started moving feature requests / ideas to Discussions.

**Comment by SurajRaika** [comment]
ya 
we should improve file component instead of creating a whole new dedicated component.

**Comment by SurajRaika** [comment]
i think file component should have option to option to select type of files.
